src/components/display/Popper.tsx

```diff
@@ -146,31 +131,25 @@ const Popper = React.forwardRef<HTMLDivElement, PopperProps>(function PopperFn(
 						bottom: virtualElement.y,
 						left: virtualElement.x,
 						y: virtualElement.y,
-						x: virtualElement.x,
-						toJSON: (): unknown => undefined
+						x: virtualElement.x
 					})
 				};
-				if (!virtualEl) {
-					popperOptions.modifiers.push({
-						name: 'flip',
-						options: {
-							fallbackPlacements: ['bottom']
-						}
+
+				if (popperRef.current) {
+					cleanup = setupFloating(virtualEl || anchorElement, popperRef.current, {
+						placement,
+						middleware: [
+							offset(8),
+							!virtualEl && flip({ fallbackPlacements: ['bottom'] }),
```

Comment:
when virtualEl is defined the second element of middleware array is false...is this the desired behaviour?
![Screenshot 2023-11-02 at 11 35 26](https://github.com/zextras/carbonio-design-system/assets/27196165/bf2ae388-908d-40ff-8b7d-d8522968dc06)

Reply:
Yes it is wanted. In the previous version the flip modifier was added only when the virtualEl was not defined. In order to keep the same behavior, we need to conditionally set that middleware in floating. We can do it with a false, since false will be filtered out (see https://floating-ui.com/docs/computePosition#conditional)